app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
+  },
+  "showAdvancedOptions": {
+    "message": "Mostrar Opciones Avanzadas"
+  },
+  "hideAdvancedOptions": {
+    "message": "Ocultar Opciones Avanzadas"
+  },
+  "optionalChainId": {
+    "message": "ChainID (opcional)"
+  },
+  "optionalSymbol": {
+    "message": "Símbolo (opcional)"
+  },
+  "optionalNickname": {
+    "message": "Apodo (opcional)"
+  },
+  "newTotal": {
+    "message": "Nuevo Total"
+  },
+  "newTransactionFee": {
+    "message": "Nueva Comisión por Transacción"
+  },
+  "noConversionRateAvailable": {
+    "message": "No Hay Tasa de Conversión"
+  },
+  "notFound": {
+    "message": "No se localizó"
+  },
+  "noWebcamFoundTitle": {
+    "message": "No se localizó al webcam"
+  },
+  "noWebcamFound": {
+    "message": "No se localizó al webcam de tu ordenador. Favor de volver a intentar."
+  },
+  "ofTextNofM": {
+    "message": "de"
+  },
+  "onlySendTokensToAccountAddress": {
+    "message": "Mandar solamente $1 a la dirección de una cuenta de Ethereum.",
+    "description": "muestra el símbolo del token"
+  },
+  "openInTab": {
+    "message": "Abrir en una pestaña"
+  },
+  "orderOneHere": {
+    "message": "Comprate un Trezor o Ledger y guarda tus fondos en almacenamiento frío"
+  },
+  "origin": {
+    "message": "Orígen"
+  },
+  "outgoing": {
+    "message": "De salida"
+  },
+  "parameters": {
+    "message": "Parámetros"
+  },
+  "originalTotal": {
+    "message": "Total Original"
+  },
+  "password": {
+    "message": "Contraseña"
+  },
+  "pending": {
+    "message": "pendiente"
+  },  
+  "popularTokens": {
+    "message": "Tokens Corrientes"
+  },
+  "prev": {
+    "message": "Prev"
+  },
+  "primaryCurrencySetting": {
+    "message": "Moneda Principal"
+  },
+  "primaryCurrencySettingDescription": {
+    "message": "Seleccionar nativa para prioritizar el que se muestren los valores en la moneda nativa de la cadena (p.ej. ETH). Seleccionar Fíat para prioritzar el que se muestren los valores en la moneda fíat seleccionada."
+  },
+  "queue": {
+    "message": "Cola"
+  },
+  "rejectAll": {
+    "message": "Rechazar a Todos"
+  },
+  "rejectTxsN": {
+    "message": "Rechazar a transacciones de $1"
+  },
+  "rejectTxsDescription": {
+    "message": "Está al punto de rechazar transacciones de $1 en lote."
+  },
+  "reset": {
+    "message": "Reiniciar"
+  },
+  "resetAccountDescription": {
+    "message": "Reiniciar tu cuenta borrará tu historial de transacciones."
+  },
+  "restoreAccountWithSeed": {
+    "message": "Restaurar tu Cuenta con Frase Semilla"
+  },
+  "requestsAwaitingAcknowledgement": {
+    "message": "peticiones pendientes de reconocimiento"
+  },
+  "restore": {
+    "message": "Restaurar"
+  },
+  "revealSeedWordsTitle": {
+    "message": "Frase Semilla"
+  },
+  "revealSeedWordsDescription": {
+    "message": "Si en algún momento cambias de navegador o de ordenador, necesitarás esta frase semilla para acceder a tus cuentas. Guárdatela en un lugar seguro y secreto."
```

Comment:
I'd use "Guardala¨ instead of "Guardatela"

Reply:
Yup. Already changed this.